tests/ludwig/features/test_image_feature.py

```diff
@@ -36,7 +36,7 @@ def image_config():
         "pool_strides": None,
         "fc_layers": None,
         "num_fc_layers": 1,
-        "fc_size": 256,
+        "output_size": 256,
```

Comment:
these values is likely too big, let's decrease it for faster tests

Reply:
Done